### PR TITLE
LCD-11275 Add node services

### PR DESCRIPTION
**Pull request by tainguyen100**
https://issues.liferay.com/browse/LCD-11275

### CODE REVIEW

**Review by Adriverforlife — APPROVED** [review]
LGTM

**Review by anthony-chu — APPROVED** [review]

File: node-partial-builds/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# hosting-partial-builds
```
Comment (christieyoo) [inline node-partial-builds/README.md:1]: Since this is not a hosting service, can you replace `hosting` with `node`?

**Review by christieyoo — CHANGES_REQUESTED** [review]
Other than the README, LGTM

File: node-partial-builds/README.md (hunk 1 above)
Comment (tainguyen100) [reply node-partial-builds/README.md]: Woops, thought I had switched that

**Pull request merged**